extensions/extensions-docker/aissemble-hive/aissemble-hive-service/src/main/resources/docker/Dockerfile

```diff
@@ -1,32 +1,29 @@
-FROM openjdk:11-jre-slim
+ARG METASTORE_VERSION
+FROM apache/hive:${METASTORE_VERSION} as appsource
+
+FROM eclipse-temurin:17-jre-ubi9-minimal as final
```

Comment:
A: alpine is about half the size, and even the default `17-jre` (canonical) is about 20% smaller.

Reply:
Alpine isn't published for `arm64`.  Good call on the `17-jre` option, though-- I accidentally looked at the size of the Windows images initially and thought they were larger.

Reply:
Another advantage of the `17-jre` image-- currently there are fewer vulnerabilities reported.